visualization/app/codeCharta/ui/ribbonBar/ribbonBar.component.ts

```diff
@@ -14,14 +15,19 @@ type PanelSelection = "NONE" | "AREA_PANEL_OPEN" | "HEIGHT_PANEL_OPEN" | "COLOR_
 })
 export class RibbonBarComponent implements OnInit, OnDestroy {
 	panelSelection: PanelSelection = "NONE"
+	RLOC_METRIC = "rloc"
+	LOC_METRIC = "loc"
 	experimentalFeaturesEnabled$ = this.store.select(experimentalFeaturesEnabledSelector)
 	isDeltaState$ = this.store.select(isDeltaStateSelector)
 	hasEdgeMetric$ = this.store.select(edgeMetricDataSelector).pipe(map(edgeMetricData => edgeMetricData.length > 0))
-
+	isMetricCouplingHidden = false
```

Comment:
I personally would not hide it, it should just not be active by default. The user might still want to chain the metrics together, even though the used metrics are different from the ones we normally use.